### PR TITLE
Jbr myq connector

### PR DESCRIPTION
**Pull request by jberes1987**

### CODE REVIEW

File: src/MyQConnector/Connectors/MyQConnector.cs [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+
+namespace Innovatrics.SmartFace.Integrations.MyQConnectorNamespace.Connectors
```
Comment (Luke-1988) [inline src/MyQConnector/Connectors/MyQConnector.cs:11]: MyQConnectorNamespace je trochu creepy, pouzivat "Namespace" v namespace pripadne "Class" v nasve triedy je anti-pattern. Doporucil by som to zmenit na MyQConnector a potom upravit nazov triedy (a interface) na Connector a IConnector

File: src/MyQConnector/Connectors/MyQConnector.cs [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+            {
+                string email = await GetEmailFromSmartFaceAPI(watchlistMemberId);
+                string token = await AuthenticateWithMyQAPI();
+                string userInfo = await GetUserInfo(email, token);
+                string userToken = await AuthenticateUserWithMyQAPI(userInfo);
+                await UnlockPrinter(myqPrinter, myqStreamId, userToken);
+            }
+            catch (Exception ex)
+            {
+                this.logger.Error(ex, "Error occurred in OpenAsync");
```
Comment (Luke-1988) [inline src/MyQConnector/Connectors/MyQConnector.cs:59]: co sa potom deje s vynimkou? Ak open neprejde, tak aktualne to ticho zlyha, ta exception sa nepropaguje dalej. Malo by to byt:

catch (Exception ex)
            {
                this.logger.Error(ex, "Error occurred in OpenAsync"); 
                throw;
            }

File: src/MyQConnector/Service/BridgeService.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            if (cameraToMyQMapping.WatchlistExternalIds != null)
+            {
+                if (cameraToMyQMapping.WatchlistExternalIds.Length > 0 && !cameraToMyQMapping.WatchlistExternalIds.Contains(notification.WatchlistExternalId))
+                {
+                    this.logger.Warning("Watchlist {watchlistExternalId} has no right to enter through this gate {streamId}.", notification.WatchlistExternalId, notification.StreamId);
+                    return;
+                }
+            }
+
+            var MyQConnector = this.MyQConnectorFactory.Create(cameraToMyQMapping.Type);
```
Comment (Luke-1988) [inline src/MyQConnector/Service/BridgeService.cs:65]: var myQConnector

File: src/MyQConnector/Service/BridgeService.cs [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+using Innovatrics.SmartFace.Integrations.MyQConnectorNamespace.Factories;
+
+
+namespace Innovatrics.SmartFace.Integrations.MyQConnectorNamespace.Services
+{
+    public class BridgeService : IBridgeService
+    {
+        private readonly ILogger logger;
+        private readonly IConfiguration configuration;
+        private readonly IMyQConnectorFactory MyQConnectorFactory;
```
Comment (Luke-1988) [inline src/MyQConnector/Service/BridgeService.cs:21]: myQConnectorFactory (C# ma variable ako camelCase - https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/ )

File: src/MyQConnector/Connectors/MyQConnector.cs (hunk 1 above)
Comment (jberes1987) [reply src/MyQConnector/Connectors/MyQConnector.cs]: Upravene, fixnute

File: src/MyQConnector/Connectors/MyQConnector.cs (hunk 2 above)
Comment (jberes1987) [reply src/MyQConnector/Connectors/MyQConnector.cs:59]: upravene fixnute

File: src/MyQConnector/Service/BridgeService.cs (hunk 3 above)
Comment (jberes1987) [reply src/MyQConnector/Service/BridgeService.cs]: upravene fixnute

File: src/MyQConnector/Service/BridgeService.cs (hunk 4 above)
Comment (jberes1987) [reply src/MyQConnector/Service/BridgeService.cs]: dakujem za info, dalsie miesta upravene fixnute

**Review by Luke-1988 — APPROVED** [review]

**Pull request merged**